syntax = "proto3";

package hello;

service HelloService {
    rpc SayHello (HelloRequest) returns (HelloResponse);
    rpc Verify (VerifyRequest) returns (VerifyResponse);
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string greeting = 1;
}

message VerifyRequest {
    string name = 1;
}

message VerifyResponse {
    bool decision = 1;
}
